### PR TITLE
feat: virtual background: blur refs: WSC-1259 

### PR DESCRIPTION
**Pull request by yukiLu6**

### CODE REVIEW

**Review by CerDennis — CHANGES_REQUESTED** [review]
If I change video source while blur is active, the blur effect doesn't persist and others can see my background

**Review by CerDennis — CHANGES_REQUESTED** [review]
Screenshot 2024-07-08 alle 11 42 39
Screenshot 2024-07-08 alle 11 42 49

On firefox the actual image ratio is different when the blur effect is added

File: src/meetings/components/meetingActionsBar/CameraButton.tsx [hunk 1]
```diff
@@ -100,11 +124,68 @@ const CameraButton = ({
...
+	}, [blur, meetingId, setBlur]);
+
+	const dropdownList = useMemo(() => {
+		const list: DropdownItem[] = [];
+		if (canUseVirtualBackground) {
+			list.push({
+				id: 'video-effect',
+				label: videoEffectTitle,
+				disabled: true,
+				customComponent: <Text weight="bold">Video effect</Text>
```
Comment (CerDennis) [inline src/meetings/components/meetingActionsBar/CameraButton.tsx]: Missing translation

File: src/meetings/components/meetingActionsBar/CameraButton.tsx [hunk 2]
```diff
@@ -61,6 +79,11 @@ const CameraButton = ({
 	const videoOutConn = useStore((store) => store.activeMeeting[meetingId]?.videoOutConn);
 	const setSelectedDeviceId = useStore((store) => store.setSelectedDeviceId);
 	const setLocalStreams = useStore((store) => store.setLocalStreams);
+	const setBlur = useStore((store) => store.setBlur);
+	const blur = useStore((store) => getIsBackgroundBlurred(store, meetingId));
```
Comment (CerDennis) [inline src/meetings/components/meetingActionsBar/CameraButton.tsx]: change it to isBlur to be better intuitive

**Review by lisaparma — APPROVED** [review]

**Review by CerDennis — APPROVED** [review]

**Pull request merged**